06-blockchain/03-alphabet.md

```diff
@@ -1,14 +1,14 @@
 ### alphabet contract
 
-Alphabet contract is a contract deployed in NeoFS side chain\.
```

Comment:
All of `06-blockchain/03-*` files are autogenerated from  `neofs-contract` repo. Could you please move corrections there?